lib/process-services/src/lib/task-list/components/task-filters.component.ts

```diff
@@ -95,6 +109,10 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
         }
     }
 
+    isTaskRoute(filter: FilterParamsModel) : boolean {
+        return this.currentRoute.includes('task') && this.currentFilter === filter;
```

Comment:
same as for the process filter, evaluate the flag inside `.subscribe((navigationStart: NavigationStart)`